### PR TITLE
Create gene heatmap nftest

### PR DESCRIPTION
**Pull request by nschcolnicov**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - [#94](https://github.com/nf-core/nanostring/pull/94) - Added nf-tests for local subworkflows normalize and quality_control.
+- [#102](https://github.com/nf-core/nanostring/pull/102) - Added nf-tests for local module create_gene_heatmap.
```
Comment (atrigila) [inline CHANGELOG.md:11]:
```suggestion
- [#102](https://github.com/nf-core/nanostring/pull/102) - Added nf-tests for local module `CREATE_GENE_HEATMAP`.
```

File: modules/local/create_gene_heatmap/meta.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        pattern: "*ENDO.tsv"
+  - - counts:
+        type: file
+        description: |
+          A tab-separated file containing counts data for genes, used when no gene set YAML is provided.
+        pattern: "*normalized_counts.tsv"
+  - - heatmap_genes_to_filter:
+        type: file
+        description: |
+          "Path to yml file (list, one item per line) to specify which genes should be used for the gene-count heatmap."
```
Comment (atrigila) [inline modules/local/create_gene_heatmap/meta.yml:26]:
```suggestion
          Path to yml file (list, one item per line) to specify which genes should be used for the gene-count heatmap.
```

File: modules/local/create_gene_heatmap/meta.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  - bioinformatics tools
+  - data visualization
+  - gene heatmap
+tools:
+  - create_gene_heatmap:
+      description: |
+        The create_gene_heatmap module generates a heatmap of gene counts.
+        It supports gene selection via additional files and provides a log-transformed visualization of the data.
+input:
+  - - annotated_counts:
```
Comment (atrigila) [inline modules/local/create_gene_heatmap/meta.yml:13]:
```suggestion
  - annotated_counts:
```

Comment (nschcolnicov) [reply modules/local/create_gene_heatmap/meta.yml:13]: This one I think is actually correct, from looking at other implementations of the meta.yml file (i.e [MultiQC ](https://github.com/nf-core/modules/blob/master/modules/nf-core/multiqc/meta.yml) or [bwa index](https://github.com/nf-core/modules/blob/master/modules/nf-core/bwa/index/meta.yml) ). The way it is typically done, we use double hyphen for each of the input channels and a single hyphen for each subsequent item of the same channel.

**Pull request merged**